### ISSUE TITLE
[FIX] Display of Profile Cards' Carousel 用户卡片的滚动图片展示的显示问题

### ISSUE THREAD

**Issue by CTlandu** [CTlandu/PhotoBuddy#3]
If we go to a Profile Card (either "Photographer" or "Model"), there are a few issues with the Carousel display:

1. Photo size not consistent 

- Photo sizes are different if total number of display is different (e.g. 7 images total & 2 images total)
- We want to make photo size consistent no matter how many photos are displayed (1-9)

2. Carousel Logics Issue

- The carousel display is kinda messy now. Each time, only 1 image should be displayed. And we should use right/left arrows to control the rolling display. For each image, it should occupy the full width of the card 

3. The arrow buttons are not distinct (color too transparent and buttons too small)

如果我们进入一个个人资料卡（无论是“摄影师”还是“模特”），轮播图显示存在以下几个问题：

1. 图片大小不一致
    如果显示的图片总数不同，图片大小也会不同（例如，总共有 7 张图片和总共有 2 张图片时，图片大小不同）。
     我们希望无论显示多少张照片（1-9 张），照片的大小都保持一致。
2. 轮播逻辑问题
    现在轮播图的显示有些混乱。每次只应显示一张图片，应该使用左右箭头来控制滚动显示。每张图片应占据卡片的全宽。
3.   箭头按钮不明显（颜色过于透明且按钮太小）。

Files might need updates:
1. ModelCard.jsx
2. PhotographerCard.jsx
Can go directly to the "Carousel" parts starting approximately on line 175

<img width="339" alt="1724776205549" src="https://github.com/user-attachments/assets/971081b1-2619-4058-9b82-aca1294bb959">
<img width="344" alt="1724776221801" src="https://github.com/user-attachments/assets/3ec87eb4-5d62-4442-a85f-35154d57304e">

What we want: 
<img width="451" alt="848e373cc567a0107ab706637b8496a" src="https://github.com/user-attachments/assets/c2b2c935-4031-4acd-b61b-9ef7e883e173">

**Comment by BlitzenWang** [comment]
![image](https://github.com/user-attachments/assets/2d1ca76f-f9de-44b3-8788-efcf290163f7)
![image](https://github.com/user-attachments/assets/96c085d9-6645-4429-bcac-55ed38a021e8)

Image slider currently works only for model cards. We may need to standardize the variable names for both photographer and model profiles

**Comment by BlitzenWang** [comment]
Currently the left arrow is out of bounds. Will need to change the layout of its parent div for it to work.

**Comment by CTlandu** [comment]
Thank you Blitzen, the updates look awesome.

I've fixed the issues you had:
1. Now the image slider works for both model and photographer cards. I simply created 2 identical imageSlider components for both P & M. So we do not need to touch the variable names
2. The left arrow was pulled to the front now. It has low z index before so it's covered by the image. I also made the image smaller so the 2 arrows don't block it.

Great work!
<img width="350" alt="1726116853330" src="https://github.com/user-attachments/assets/da40af9b-421e-4f6d-ab0e-0a2ff7fca17a">
